THIRD_PARTY_LICENSES.txt

```diff
@@ -9770,6 +9771,11 @@ github.com/golang-migrate/migrate/v4
 Copyright (c) 2016 Matthias Kadenbach
 Copyright (c) 2018 Dale Hui
 
+-------- Dependency
+github.com/gordonklaus/ineffassign
```

Comment:
if you add this here, it needs a PLS approval.

Reply:
resolved - ran go mod tidy as you suggested, and it was removed